packages/next/src/client/components/react-dev-overlay/_experimental/internal/styles/Base.tsx

```diff
@@ -5,6 +5,8 @@ export function Base() {
   return (
     <style>
       {css`
+        @import url('https://fonts.googleapis.com/css2?family=Geist:wght@400;900&family=Geist+Mono:wght@400;900&display=swap');
+
```

Comment:
non-blocking nit: ideally, this can be bundled in so we dont need to download from google

Reply:
Maybe we could pre-download it in the taskfile so it's still updated whenever we do a release

Reply:
Will revisit to use local woff2 later!